src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html,
+body {
+  font-family: "Roboto", sans-serif;
+  /* font-weight: 500; */
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  /* height: 60px; */
+
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  cursor: pointer;
+  display: block;
+  height: 40px;
+}
+
+.nav__item {
+  text-transform: uppercase;
+  white-space: nowrap;
+  align-items: center;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+/* 12.[TESTS] */
+
+.nav__link {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  height: 60px;
+  cursor: pointer;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
```

Comment:
it is better to use CSS variable not to repeat the color literal

Reply:
also done.